### PR TITLE
add tabs component

### PR DESCRIPTION
**Pull request by jivey**
for: https://github.com/openstax/unified/issues/2419

### CODE REVIEW

**Comment by jivey** [comment]
Adding @TomWoodward and @Dantemss too in case you all have thoughts about my use of the react-aria-components library.

File: src/components/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    props.variant === "button-bar" ? buttonBarCss : tabsCss}
+`;
+
+export const Tabs = ({
+  variant,
+  size = "large",
+  className,
+  children,
+  ...restProps
+}: TabsProps & AC.TabsProps) => {
```
Comment (Dantemss) [inline src/components/Tabs.tsx:127]: TabsProps already includes AC.TabsProps

**Review by Dantemss — COMMENTED** [review]
The usage seems fine to me? I don't have strong opinions about it.

File: src/components/Tabs.tsx (hunk 1 above)
Comment (jivey) [reply src/components/Tabs.tsx]: Good catch, forgot I moved it up there

**Comment by jivey** [comment]
@bethshook I realized I needed the button-bar look, without the tab functionality, so I extracted that into a component that uses the same styles (so we can still have tabs with the button-bar styles) if you want to take another look.

**Comment by bethshook** [comment]
looks good. only thing I noticed is that in the button bar story the selected button doesn't change on click.

**Review by bethshook — APPROVED** [review]

**Pull request merged**